src/Adapter/MSTest.CoreAdapter/Properties/TypesToLoadAttribute.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+using Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.Utilities;
+
+[assembly: TypesToLoad(typeof(MSTestDiscoverer), typeof(MSTestExecutor))]
+
+namespace Microsoft.VisualStudio.TestPlatform.ObjectModel.Utilities
+{
+    /// <summary>
+    /// Custom Attribute to specify the exact types which should be loaded from assembly
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Assembly, Inherited = false, AllowMultiple = false)]
+    internal sealed class TypesToLoadAttribute : Attribute
```

Comment:
I'd [mentioned](https://github.com/microsoft/testfx/pull/746#discussion_r533752933) this to @Sanan07, but perhaps `TestExtensionTypesAttribute` would be be a better name (i.e. a bit less confusing for adapter implementers).

Reply:
That's a good idea and a relatively simple change. I'll do it tomorrow morning.